aspnet/conf.py

```diff
@@ -99,7 +99,10 @@
 
 rst_prolog = """
 .. raw:: html
-
+	
+	<div id="helpful-position"></div><div id="helpful" class="helpfulness-container fixed" style="display:none;"><div class="helpfulness"><div class="helpfulness-form"><h2>Was this page helpful?</h2><span>Your feedback about this content is important. Let us know what you think.</span><a id="helpfulness-btn-yes" href="javascript:;" class="common-btn first">Yes</a><a id="helpfulness-btn-no" href="javascript:;" class="common-btn">No</a></div><div class="helpfulness-form-no" style="display:none;"><h2>Was this page helpful?</h2>
+	Sorry this wasn't helpful.
+	<input type="text" id="txt-helpfulness" value="Please let us know why this wasn't helpful" data-uitext="Please let us know why this wasn't helpful" data-uitype="autoreset" maxlength="160"><span id="helpfulness-characters-left">160 characters remaining</span><a id="helpfulness-btn-submit" href="javascript:;" class="common-btn">Submit</a><a id="helpfulness-btn-skip" href="javascript:;" class="common-btn secondary">Skip this</a></div><a href="javascript:;" class="helpfulness-close">✖</a><div class="messages processing" style="display:none;">Sending feedback...</div><div class="messages success" style="display:none;">Thank you for your feedback!</div><div class="messages error" style="display:none;">Error during submission!</div></div></div>
```

Comment:
Agreed. The only reason we are doing it without the html5 placeholder is because that is how asp.net was built. We wanted to port this functionality from asp.net to docs.asp.net and I wanted to keep the codebases in line. If this is a problem I can rework the asp.net site to use the placeholder attribute and add the necessary fallback for older browsers.